src/test/java/core/basesyntax/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,96 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Storage;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyImplTest {
```

Comment:
this violates the concept of unit testing, it is better to test each handler separately